internal/dao/stubs/account_dao.go

```diff
@@ -0,0 +1,79 @@
+package stubs
+
+import (
+	"context"
+	"errors"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/RHEnVision/provisioning-backend/internal/pointer"
+)
+
+var notInStoreError = errors.New("The Account you've been looking for does not exist")
```

Comment:
By capitalization I mean the string, not the variable ;) I think this actually must be exported. Also maybe you can move it to some common filename like `errors.go` or something like that because these should be shared across DAOs.

```suggestion
var StubNotFoundError = errors.New("DAO record does not exist")
```

Reply:
done :)